code/modules/hippie/code/modules/reagent/reagent_dispenser.dm

```diff
@@ -3,7 +3,7 @@
 		if(!user.drop_item())
 			return
 		qdel(I)
-		paper_cups++
+		cups++
```

Comment:
Too bad the var is still called paper_cups, and now you are just trying to something completely different that isn't even defined.